src/OrbitGl/App.cpp

```diff
@@ -1113,6 +1113,25 @@ static std::unique_ptr<CaptureEventProcessor> CreateCaptureEventProcessor(
                                     orbit_client_model::capture_serializer::GenerateCaptureFileName(
                                         process_name, absl::Now(), "_autosave");
 
+  uint64_t suffix_number = 0;
+  while (true) {
+    auto file_exists_or_error = orbit_base::FileExists(file_path);
+    if (file_exists_or_error.has_error()) {
+      ERROR("Unable to stat \"%s\": %s", file_path.string(),
```

Comment:
I think this should be "Unable to check for existence of". (`stat` would return an error if the file doesn't exist, but this is about having an error on the check for existence itself)

Reply:
done.